models/autoevent.go

```diff
@@ -16,10 +16,21 @@ package models
 
 import "encoding/json"
 
+/*
+ * This file is the model for the AutoEvent object in EdgeX
+ *
+ * AutoEvent struct
+ */
 type AutoEvent struct {
-	Frequency int32  `json:"frequency,omitempty"`
-	OnChange  bool   `json:"onChange,omitempty"`
-	Resource  string `json:"resource,omitempty"`
+	// Frequency indicates that how often the specific resource need to be polled.
+	// It represents the number of events per day.
+	// For example, 86400 would be once per second, 24 would be onece per hour etc.
+	Frequency uint32 `json:"frequency,omitempty"`
+	// OnChange indicates whether generate the events only if the reading value is different from the previous one.
```

Comment:
"`OnChange indicates whether the device service will generate an event only if the reading value is different from the previous one.`"